apiserver/facades/client/sshclient/register.go

```diff
@@ -35,8 +35,14 @@ func newFacade(ctx facade.Context) (*Facade, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	bck := backend{
+		State:               st,
+		EnvironConfigGetter: stateenvirons.EnvironConfigGetter{Model: m},
+		controllerTag:       m.ControllerTag(),
+		modelTag:            m.ModelTag(),
+	}
 	return internalFacade(
-		&backend{st, stateenvirons.EnvironConfigGetter{Model: m}, m.ModelTag()},
+		&bck,
```

Comment:
nit - we tend to avoid abbreviations like `bck`

Reply:
Easy fix.